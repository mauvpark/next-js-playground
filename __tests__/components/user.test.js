import renderer from "react-test-renderer";
import User from "../../components/user";
import userListData from "../../__mocks__/user-list.mock";

it("<User /> should render userList mock datas' index 0", () => {
	const tree = renderer.create(<User data={userListData()[0]} />).toJSON();
	expect(tree).toMatchSnapshot();
});
