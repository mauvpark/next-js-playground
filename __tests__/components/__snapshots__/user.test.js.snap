// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User /> should render userList mock datas' index 0 1`] = `
<div>
  <h2>
    Bret
  </h2>
  <p>
    hildegard.org
  </p>
</div>
`;
